src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -542,7 +555,7 @@ export const StakeInput: React.FC<StakeInputProps & StakeRouteProps> = ({
               isValidWallet={Boolean(isChainSupportedByWallet || isAccountMetadataLoading)}
               isDisabled={Boolean(
                 errors.amountFieldInput ||
-                  !runeAddress ||
+                  (!runeAddress && !currentRuneAddress) ||
```

Comment:
It was the reason why the stake button was disabled, we don't manually set the runeAddress all the time